src/resources/assets/js/geo/components/geoMapModal.vue

```diff
@@ -1,27 +1,25 @@
 <template>
     <modal 
-        v-model="showModal" 
-        title="Map Filter" 
-        ok-text="Add rule"
-        cancel-text="Cancel"
-        ok-type="default"
-        cancel-type="default"
+        v-model="show"
+        title="Map Filter"
         size="lg"
-        v-on:hide="callback"
       >
         <p v-html="text"></p>
```

Comment:
Please show the text below the map and with `class="text-muted"`.

Reply:
resolved 2c1d96e01657c8566d42cef264557909bf4a0108